src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -1106,7 +1106,7 @@ describe('Step Packages', () => {
     searchboxChosen.click();
     await user.type(searchboxChosen, 'asdf');
 
-    expect(screen.getAllByText('No packages found').length === 2);
+    expect(screen.getByText('No packages found')).toBeInTheDocument();
```

Comment:
I'm not sure this is testing the same thing... "No packages found" should appear twice, that's why we check whether length === 2 on getAllByText.

Reply:
Disregard above, it should only appear once.